lib/tasks/hydranorth.rake

```diff
@@ -261,4 +261,27 @@ namespace :hydranorth do
     end
   end    
 
+  desc 'Quickstart for getting a fresh hydranorth development install ' +
```

Comment:
This is more for discussion but what's your opinion on limiting your lines? (I assume this is 80 characters which is the standard in most software communities)?

I personally prefer around 100 (https://github.com/airbnb/ruby/blob/master/rationales.md#line-length) as that's a good size on a modern screen before it start's scrolling. But regardless in this project we have no enforcement on it (no rubocop).  So up to you if you want to concat strings like this. Just curious, be good to know what other's think about this (especially when we nail down these rules in https://github.com/ualbertalib/Hydranorth2/issues/55)?

Also to be nitpicky :stuck_out_tongue_winking_eye: , it should probably be using  a backslash for continuation (`\`)? instead of actual true string concatenation (`+`)?
```ruby
desc 'Quickstart for getting a fresh hydranorth development install '\
 'up and running'
```